src/lib/core/typography/_typography.scss

```diff
@@ -23,7 +23,9 @@
   $body-2:      mat-typography-level(14px, 24px, 500),
   $body-1:      mat-typography-level(14px, 20px, 400),
   $caption:     mat-typography-level(12px, 12px, 400),
-  $button:      mat-typography-level(14px, 14px, 500)
+  $button:      mat-typography-level(14px, 14px, 500),
+  // Line-height must be unit-less fraction of the font-size.
+  $input:       mat-typography-level(16px, 1.125, 400)
```

Comment:
@jelbourn @crisbeto spec says `16px` for inputs, but by doing this I ruin its ability to inherit from its parent (e.g. so that an input inside an `<h1>` matches the font size, should I keep this as `16px` or change to `inherit`?